lib/plausible/stats/filters/where_builder.ex

```
@@ -150,6 +152,15 @@ defmodule Plausible.Stats.Filters.WhereBuilder do
     true
   end
 
+  defp add_filter(table, _query, filter) do
+    Logger.info("Unable to process garbage filter. No results are returned",
```

At first glance I'm not a fan of silently accepting/ignoring garbage. Would it make sense to error out the whole query here due to invalid filter?

---

Not a fan either but there's an existing test that tested against sentry spam when security scanners are run against our API: test/plausible_web/controllers/api/stats_controller/conversions_test.exs:211. 

Not sure if this is indeed the best solution - do you recommend keeping this, removing the test or something else.

---

>  do you recommend keeping this, removing the test or something else.

Can we make it a 400 Bad Request rather than 200 OK?